src/js/entities-service/entities/patient-search-results.js

```diff
@@ -47,7 +47,9 @@ const Collection = BaseCollection.extend({
     const filter = { search };
 
     delete this._hasIdentifiers;
-    this.fetcher = this.fetch({ data: { filter } });
+    this.controller.abort();
+    this.controller = new AbortController();
+    this.fetcher = this.fetch({ data: { filter }, signal: this.controller.signal });
```

Comment:
we need a custom abort controller for this request as we're only aborting under certain circumstances.